code/datums/skills/_skill.dm

```diff
@@ -4,3 +4,50 @@
 
 /datum/skill/proc/get_skill_speed_modifier(level)
 	return
+
+/datum/skill/proc/get_skill_effciency_modifier(level)
```

Comment:
Unused? Remove this if you dont plan to use it, I plan to do something else with skill-modifiers that has to do with assoc lists. (key "string-of-modifier", - value list of modifiers by level)

Reply:
nightcoding ™️ 

was going to separate speed and efficacy but just put it under the same guy

Reply:
efficacy

Reply:
am r*